.github/workflows/deploy-worker.yml

```diff
@@ -115,3 +122,23 @@ jobs:
           wranglerVersion: ${{ env.wranglerVersion }}
           workingDirectory: ${{ github.event.inputs.directory }}
           command: deploy --no-bundle --name=${{ github.event.inputs.appId }} --dispatch-namespace ${{ github.event.inputs.dispatchNamespace }} ${{ steps.get-script.outputs.WORKER_SCRIPT }}
+
+  attest:
```

Comment:
Making this a separate job creates a second set of webhook events
https://github.com/codius/codius-workers/blob/84c9cd53efd2eddc69f5fae7d51bd1e0295d9ef3/packages/codius-astro/src/pages/webhooks/github/workflow-job.ts#L11
https://github.com/codius/codius-workers/blob/84c9cd53efd2eddc69f5fae7d51bd1e0295d9ef3/packages/codius-astro/src/pages/webhooks/github/workflow-job.ts#L27